DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -247,7 +313,7 @@ function Set-TargetResource
                     $NewWebsiteSplat.Add($_.Key, $_.Value)
                 }
 
-                # If there are no other websites, specify the Id parameter for the new website.
+                # If there are no other websites, specify the Id Parameter for the new website.
```

Comment:
Capitalizing `Parameter` isn't necessary here. Seems to be a Ctrl-H result. Again.